samples/compiler_plugins/xnnpack/src/xnnpack/Conversion/StablehloToXnnpack.cpp

```diff
@@ -14,6 +20,40 @@
 #include "xnnpack/Conversion/Passes.h.inc"
 
 namespace mlir::iree_compiler::IREE::Xnnpack {
+
+static llvm::cl::opt<std::string> patternFileName(
```

Comment:
We can specify this also a an option of the pass itself, to make it explicit that this pass is the one that uses it. Here's an example of how I did this in Torch-MLIR

https://github.com/ramiro050/torch-mlir/commit/c3455de540671e098ccb1e2604a1272d29a6f7b4#diff-82d50f7a1c3af6985b1aee2f901e35dcb8bd3df12f6009c11bb17cebfb2ddbb0

Reply:
Thanks for the info! This seems to need more work since I need to set up the compiler or plugin level option to pass the info to the pass when calling `createConvertStablehloToXnnpackPass`. I will keep this as is.